src/cc/usdt/usdt.cc

```diff
@@ -353,7 +383,8 @@ void Context::each_uprobe(each_uprobe_cb callback) {
 
     for (Location &loc : p->locations_) {
       callback(loc.bin_path_.c_str(), p->attached_to_->c_str(), loc.address_,
-               pid_.value_or(-1));
+               pid_.value_or(
+                   -1));  // FIXME ADD semaphore offset to this tuple needed?
```

Comment:
i don't think that the ref_ctr nedes to be added to this tuple, is there any reason to?

Reply:
This seems like this used by the python API